avm/utilities/pipelines/platform/Set-AvmGitHubPrLabels.ps1

```diff
@@ -0,0 +1,73 @@
+﻿<#
+.SYNOPSIS
+Checks new PRs and adds labels, depending if module owner can approve, or core team is needed
+
+.DESCRIPTION
+For the given PR, the "needs core team" label will be added, if the module is orphaned or the sole module owner is creating the PR
+
+.PARAMETER Repo
+Mandatory. The name of the respository to scan. Needs to have the structure "<owner>/<repositioryName>", like 'Azure/bicep-registry-modules/'
+
+.PARAMETER RepoRoot
+Optional. Path to the root of the repository.
+
+.PARAMETER PrUrl
+Mandatory. The URL of the GitHub pull request, like 'https://github.com/Azure/bicep-registry-modules/pull/2540'
+
+.EXAMPLE
+Set-AvmGitHubPrLabels -Repo 'Azure/bicep-registry-modules' -PrUrl 'https://github.com/Azure/bicep-registry-modules/pull/2540'
+
+.NOTES
+Will be triggered by the workflow avm.platform.set-avm-github-pr-labels.yml
```

Comment:
Suggest to stay generic to avoid getting stale (it's already outdated name)
```suggestion
Will be triggered by the corresponding platform workflow
```